.gitmodules

```diff
@@ -1,15 +1,13 @@
 [submodule "crossbeam-ebr"]
 	path = crossbeam-ebr
-	url = https://github.com/kaist-cp/crossbeam
+	url = https://github.com/powergee/crossbeam.git
+	branch = kaist-no-nested
 [submodule "crossbeam-pebr"]
 	path = crossbeam-pebr
 	url = https://github.com/kaist-cp/crossbeam
 [submodule "nbr-rs"]
 	path = nbr-rs
 	url = https://github.com/powergee/nbr-rs.git
-[submodule "cdrc-rs"]
```

Comment:
why we don't need this?

Reply:
`cdrc-rs` was embedded in the repository during the development of CIRC, and its intention was to facilitate the development process. However, as we have finished it, it will be better to make it a separate submodule.

I have replaced it with the dedicated submodule.